mutis/signal.py

```diff
@@ -21,16 +21,16 @@
 class Signal:
     """Analysis and generation of a signal.
 
-    Description goes here.
+    Class for a generic signal.
 
-    Parameters
+    Attributes
     ----------
     times : :class:`numpy.ndarray` or :class:`pandas.Series`
         Values of the time axis.
     values : :class:`numpy.ndarray` or :class:`pandas.Series`
         Values of the signal axis.
     fgen : :class:`str`
-        Method used to generate the synthetic signal.
+        Method to generate the synthetic signal.
```

Comment:
I would add other attributes missing in this list ?

Reply:
Done.